### PR TITLE
Waiting for cert-manager upgrade then update the operandconfig

### PR DESCRIPTION
**Pull request by horis233**

### CODE REVIEW

File: controllers/bootstrap/init.go [hunk 1]
```diff
@@ -212,7 +220,15 @@ func (b *Bootstrap) InitResources(manualManagement bool) error {
 		return err
 	}
 
-	if err := b.createOrUpdateResources(annotations, OdlmCrResources); err != nil {
+	if err := b.createOrUpdateResources(annotations, RegistryCrResources); err != nil {
```
Comment (Daniel-Fan) [inline controllers/bootstrap/init.go:223]: Do we need to remain the second parameter as the string slice in this case?
or we could use func `createOrUpdateResource` if it is only the string.

File: controllers/bootstrap/init.go [hunk 2]
```diff
@@ -497,13 +531,53 @@ func (b *Bootstrap) waitOperatorReady(name, namespace string) error {
...
 			}
-			if csv.Status.Phase == olmv1alpha1.CSVPhaseSucceeded {
-				return true, nil
+			if csv.Status.Phase != olmv1alpha1.CSVPhaseSucceeded {
+				return false, nil
+			}
+			if csv.Status.Reason != olmv1alpha1.CSVReasonInstallSuccessful {
+				return false, nil
 			}
+			klog.Errorf("Cluster Service Version %s/%s is ready", csv.Namespace, csv.Name)
```
Comment (Daniel-Fan) [inline controllers/bootstrap/init.go:540]: Should we use `klog.infof` instead of `errorf`?

File: controllers/bootstrap/init.go (hunk 1 above)
Comment (horis233) [reply controllers/bootstrap/init.go]: Nice catch, updated

File: controllers/bootstrap/init.go (hunk 2 above)
Comment (horis233) [reply controllers/bootstrap/init.go]: Nice catch, updated

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**